internal/services/engines/jvm/samples_test.go

```diff
@@ -13,3 +13,53 @@
 // limitations under the License.
 
 package jvm
+
+const (
+	SampleVulnerableHSJVM38 = `
+class T {
+	void f() {
+		String input = "test input";
+		Base64.getEncoder().encodeToString(input.getBytes());
+
+		Base64 base64 = new Base64();
+		String encodedString = new String(base64.encode(input.getBytes()));
+	}
+}
+	`
+
+	SampleVulnerableHSJVM24 = `
+class T {
+	void f(String value) {
+		byte[] decodedValue = Base64.getDecoder().decode(value);
+	}
+}
+	`
+)
+
+const (
+	SampleSafeHSJVM38 = `
+class T {
+	void f() {
+		obj.addContentType("application/x-www-form-urlencoded")
+	}
+}
+	`
+	SampleSafeHSJVM382 = `
```

Comment:
Change it to `Sample2SafeHSJVM38` to be more readable

Reply:
Fixed.